test_libs/pyspec/eth2spec/utils/ssz/ssz_impl.py

```diff
@@ -92,12 +97,11 @@ def encode_series(values: Series):
 def pack(values: Series):
     if isinstance(values, bytes):  # Bytes and BytesN are already packed
         return values
-    elif isinstance(values, Bitvector):
-        as_integer = sum([values[i] << i for i in range(len(values))])
-        return as_integer.to_bytes((values.length + 7) // 8, "little")
-    elif isinstance(values, Bitlist):
-        as_integer = sum([values[i] << i for i in range(len(values))])
-        return as_integer.to_bytes((values.length + 7) // 8, "little")
+    elif isinstance(values, Bitvector) or isinstance(values, Bitlist):
```

Comment:
```suggestion
    elif isinstance(values, (Bitvector, Bitlist)):
```

And can you confirm that we are not prefixing the bit here, since we already mix in the length for hashing?

Reply:
Correct, the Merkleization does not mix in the bit delimiter.